pkg/security/ebpf/c/activity_dump.h

```diff
@@ -195,16 +195,34 @@ __attribute__((always_inline)) void cleanup_traced_state(u32 pid) {
 #define NO_ACTIVITY_DUMP       0
 #define IGNORE_DISCARDER_CHECK 1
 
+struct ad_container_id_comm {
+    char container_id[CONTAINER_ID_LEN];
+    char comm[TASK_COMM_LEN];
+};
+
+struct bpf_map_def SEC("maps/activity_dump_buffer") activity_dump_buffer = {
```

Comment:
nit: I like the `*_gen` (so `ad_container_id_comm_gen` in this case) convention for maps that are used just to generate objects :)

Reply:
Oh yeah, I can change that